public/pages/DetectorConfig/containers/Features.tsx

```diff
@@ -0,0 +1,258 @@
+      {
+        field: 'definition',
+        name: 'Feature definition',
+        render: (featureIndex: number) => {
+          const feature = featureAttributes[featureIndex];
+
+          const metaData = get(
+            this.props.detector,
+            `uiMetadata.features.${feature.featureName}`,
+            {}
+          );
+
+          if (
+            Object.keys(metaData).length === 0 ||
+            metaData.featureType == FEATURE_TYPE.CUSTOM
+          ) {
+            return (
+              <div>
+                <p>
+                  Custom expression:{' '}
+                  <EuiLink
+                    data-test-subj={`viewFeature-${featureIndex}`}
+                    onClick={() => this.showModal(featureIndex)}
+                  >
+                    View code
+                  </EuiLink>
+                </p>
+
+                {!this.getModalVisibilityChange(featureIndex) ? null : (
+                  <CodeModal
+                    code={JSON.stringify(feature.aggregationQuery, null, 4)}
+                    title={feature.featureName}
+                    subtitle="Custom expression"
+                    closeModal={() => this.closeModal(featureIndex)}
+                    getModalVisibilityChange={() =>
+                      this.getModalVisibilityChange(featureIndex)
+                    }
+                  />
+                )}
+              </div>
+            );
+          } else {
+            return (
+              <div>
+                <p>Field: {metaData.aggregationOf || ''}</p>
+                <p>Aggregation method: {metaData.aggregationBy || ''}</p>
+              </div>
+            );
+          }
+        },
+      },
+      {
+        field: 'state',
+        name: 'State',
+      },
+    ];
+
+    const getCellProps = () => {
+      return {
+        className: 'featureText',
+        textOnly: true,
+      };
+    };
+
+    const handleOnCreate = () => {
+      // TODO: add logic
+      console.log('to be implemented');
+    };
+
+    const featureNum = Object.keys(featureAttributes).length;
+
+    return (
+      <ContentPanel
+        title={`Features (${featureNum})`}
+        titleSize="s"
+        subTitle={
+          <p>
+            Specify index fields that you want to find anomalies for by defining
+            features. A detector can discover anomalies across up to 5 features.
+            Once you define the features, you can preview your anomalies from a
+            sample feature output.{' '}
+            <EuiLink
+              href="https://github.com/opendistro-for-elasticsearch/anomaly-detection"
+              target="_blank"
+            >
+              Learn more &nbsp;
+              <EuiIcon size="s" type="popout" />
+            </EuiLink>
+          </p>
+        }
+        subTitleClassName="fieldsSubtitle"
+        actions={[
+          <EuiButton
+            href={`${PLUGIN_NAME}#/detectors/${this.props.detectorId}/features`}
+          >
+            Edit
```

Comment:
Add confirm modal for running detector. You can refer to https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/pages/DetectorDetail/containers/DetectorDetail.tsx#L322

Reply:
Similar to edit detector button.
BTW, we should not show side nav bar on create/edit detector page.  Add this line: [`useHideSideNavBar(true, false);`](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/pages/DetectorDetail/containers/DetectorDetail.tsx#L101) to [CreateDetector.tsx line 64](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/pages/createDetector/containers/CreateDetector.tsx#L64)

Reply:
Fix breadcrumb of [CreateDetector.tsx](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/pages/createDetector/containers/CreateDetector.tsx#L70-L79) like this
```
//Set breadcrumbs based on Create / Update
  useEffect(() => {
    const createOrEditBreadcrumb = props.isEdit
      ? BREADCRUMBS.EDIT_DETECTOR
      : BREADCRUMBS.CREATE_DETECTOR;
    let breadCrumbs = [
      BREADCRUMBS.ANOMALY_DETECTOR,
      BREADCRUMBS.DETECTORS,
      createOrEditBreadcrumb,
    ];
    if (detector && detector.name) {
      breadCrumbs.splice(2, 0, {
        text: detector.name,
        href: `#/detectors/${detectorId}`,
      });
    }
    chrome.breadcrumbs.set(breadCrumbs);
  });
```

Reply:
all of the above has been addressed by you or by me.  Thanks.